docs/streams.html

```diff
@@ -398,6 +401,84 @@
         in <code>StreamsConfig</code> before running it. A complete list of
         Kafka Streams configs can be found <a href="#streamsconfigs"><b>here</b></a>.
         </p>
+
+        <h2><a id="streams_upgrade" href="#upgrade">Upgrade guide and API changes</a></h2>
+
+        <h3><a id="streams_upgrade_1020" href="#upgrade_1020">Upgrading a Kafka Streams Application</a></h3>
+
+        <h4>Upgrading from 0.10.1.x to 0.10.2.0</h4>
+        <p>
+        See <a href="../#upgrade_1020_streams">Upgrade Section</a> for details.
+        </p>
+
+        <h3><a id="streams_api_changes" href="#api_changes">Streams API changes in 0.10.2.0</a></h3>
+        <li> New methods in <code>KafkaStreams</code>:
+            <ul>
+                <li> set a listener to react on application state change via <code>#setStateListener(StateListener listener)</code> </li>
+                <li> retrieve the current application state via <code>#state()</code> </li>
+                <li> retrieve the global metrics registry via <code>#metrics()</code> </li>
+                <li> apply a timeout when closing an application via <code>#close(long timeout, TimeUnit timeUnit)</code> </li>
+                <li> specify a custom indent when retrieving Kafka Streams information via <code>#toString(String indent)</code> </li>
+            </ul>
+        </li>
+        <li> Parameter updates in <code>StreamsConfig</code>:
+            <ul>
+                <li> parameter "zookeeper.connect" was deprecated </li>
```

Comment:
We should add why -- because now Streams no longer interacts with ZK.  The only reason we still tolerate the zookeeper.connect setting is for user convenience (their apps will not fail to start only because they haven't yet removed the unused ZK setting).